### PR TITLE
Replace GfxDrawString() with DrawText()

### PR DESCRIPTION
**Pull request by Harry-Hopkinson**
Replace GfxDrawString() with DrawText() for https://github.com/OpenRCT2/OpenRCT2/issues/21992

### CODE REVIEW

**Comment by Gymnasiast** [comment]
Interesting error that MSVC is showing there... maybe including Text.h helps (where the function is defined).

**Comment by duncanspumpkin** [comment]
> I think I see what is happening... it looks like there is a WinAPI function called `DrawText` which is overwriting the OpenRCT2 function. Should I change the name of the function?

i've a PR that will fix the issue. The windows api shouldn't have been pulled into that file.

**Comment by Gymnasiast** [comment]
@Harry-Hopkinson That PR has now been merged, could you rebase this one? You don’t need to touch any of your code - also leave the Text.h imports as they are now (as .cpp files should always import the functions they’re using anyway).

File: src/openrct2-ui/windows/NetworkStatus.cpp [hunk 1]
```diff
@@ -9,7 +9,6 @@
 
 #include <openrct2-ui/interface/Widget.h>
 #include <openrct2-ui/windows/Window.h>
-#include <openrct2/drawing/Drawing.h>
```
Comment (Gymnasiast) [inline src/openrct2-ui/windows/NetworkStatus.cpp:12]: This file is missing a Text.h include.

Comment (Harry-Hopkinson) [reply src/openrct2-ui/windows/NetworkStatus.cpp:12]: There was no errors so I changed it.

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**